packages/ses/NEWS.md

```diff
@@ -1,5 +1,9 @@
 User-visible changes in `ses`:
 
+# Unreleased
```

Comment:
```suggestion
# Next version
```

There’s a note in CONTRIBUTING: in the release process, I grep for `# Next`. I (or whomever pulls the crank) could also grep for `# Unreleased` if you want to update the release process.

Reply:
just adopting a previous pattern, will update